pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -222,7 +222,7 @@ func initStatusServer(
 	agent *istio_agent.Agent,
 	shutdown context.CancelFunc,
 ) error {
-	o := options.NewStatusServerOptions(proxy, proxyConfig, agent)
+	o := options.NewStatusServerOptions(proxy.IsIPv6(), proxy.Type, proxyConfig, agent)
```

Comment:
Are we only trying to slim the library, not the cmd itself? If so, why? If not, how does it help to just make the libraries slim?

Reply:
I can revert this change. I can't finish changing pilot-agent because of that SDS import. Yes, I tried to move `model.Proxy` into `pkg/model` but that causes too much of a change if we embed a struct. So I think it's better to copy just the functions we need.